doc/how_to/observers/ipywidget.md

```diff
@@ -0,0 +1,186 @@
+# Observe ipywidgets
```

Comment:
The tests fail because this file is not an `index.md` file. But I can't see how that can make sense because later if/ when more observers are added the link will break. I need help.

![image](https://github.com/holoviz/panel/assets/42288570/4e2b29a2-d21a-4919-9003-306f53f61c15)